applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,337 @@
+
+      .. code-block:: console
+
+         west build -b nrf54l15dk_nrf54l15_cpuapp@soc1 -- -DSHIELD="peripheral_sensor_node"
+
+   #. Flash the sample using the standard |NCS| flash command:
+
+      .. code-block:: console
+
+         west flash
+
+      The command flashes all required binaries to the PDK from the build directory, then resets and runs the application.
+
+      .. note::
+
+            When programming the device, you might get an error similar to the following message::
+
+               ERROR: The operation attempted is unavailable due to readback protection in
+               ERROR: your device. Please use --recover to unlock the device.
+
+            This error occurs when readback protection is enabled.
+            To disable the readback protection, you must *recover* your device.
+
+            Enter the following command to recover the core::
+
+               west flash --recover
+
+            The ``--recover`` command erases the flash memory and then writes a small binary into the recovered flash memory.
+            This binary prevents the readback protection from enabling itself again after a pin reset or power cycle.
+
+
+Application logs
+================
+
+The application provides logs through the UART of the PDK.
+
+.. _nrf_peripheral_sensor_node_app_testing:
+
+Testing with the PDK
+====================
+
+After programming the application, perform the following steps to test the nRF Peripheral Sensor node application on the Development Kit:
+
+1. Turn on the development kit.
+   The application starts Bluetooth advertising.
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. Connect to the device from the application.
+   When connected, go to more options and click :guilabel:`Request MTU`.
+   Set the value to ``498``.
+   The device is advertising as ``Peripheral Sensor Node``.
+   The services of the connected device are shown.
+   If the device cannot be found, the PDK might be connected to another device.
+   Terminate the connection with the other device, refresh the scanning and connect to the PDK.
+   The bonding data can be erased by running the following command:
+
+      .. code-block:: console
+
+         west flash --erase
+
+#. Find the ``Env read`` charateristic (UUID ``de550004-acb6-4c73-8445-2563acbb43c2``).
+#. Enable notification for this characteristic.
+#. Find ``Env read req`` charateristic (UUID ``de550002-acb6-4c73-8445-2563acbb43c2``).
+#. Write to ``Env read req`` a number of requested history depth as little endian uint16_t.
+#. Observe the returned data in ``Env read``.
+#. See the latest measurement results of IMU and environmental sensors through a separate **Nordic Status Message Service** characteristics.
+#. To observe the logs, connect to the development kit with a terminal emulator (for example, PuTTY).
+   See :ref:`putty` for the required settings.
```

Comment:
PuTTY should not be recommended as the default terminal emulator. Please use nRF Connect Serial Terminal and link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize.html .
Re: https://github.com/nrfconnect/sdk-nrf/pull/13663